docs/configuration.md

```diff
@@ -7,17 +7,20 @@ title: Configuration
-* `EVENT_MARKER_COM_NAME` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The com name of the event marker (e.g. `COM3`). If not set, it will use the `comName` set in `public/config/trigger.js`.  If the productID is set (not an empty string), this field will be ignored.
-* `REACT_APP_VOLUME`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being asked to adjust volume. This can be used on both the desktop (electron) and online settings. Not defining this variable is equivalent to setting it to `false`. 
-* `REACT_APP_VIDEO`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the participant is being video recorded. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
-* `REACT_APP_USE_EVENT_MARKER`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-runtime-purple)![](https://img.shields.io/badge/-buildtime-blue): whether the event marker/EEG is available. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
-* `REACT_APP_USE_PHOTODIODE`![](https://img.shields.io/badge/-boolean-lightgrey)![](https://img.shields.io/badge/-buildtime-blue): whether the photodiode is in use. This can be used when the task is running as a desktop app (electron). Not defining this variable is equivalent to setting it to `false`. Build-time and run-time values need to be the same.
-* `REACT_APP_PARTICIPANT_ID` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The default participant id to show when authorizing a participant.  If not set, no default is shown (blank input box).
-* `REACT_APP_STUDY_ID` ![](https://img.shields.io/badge/-string-grey)![](https://img.shields.io/badge/-optional-yellow)![](https://img.shields.io/badge/-runtime-purple): The default study id to show when authorizing a participant.  If not set, no default is shown (blank input box).
+* `ELECTRON_START_URL` ![string](https://img.shields.io/badge/-string-grey)![optional](https://img.shields.io/badge/-optional-yellow): URL (e.g. `http://localhost:3000`) where the front end of the app is being hosted - also used in `electron.js` to indicate the app is running in dev mode
```

Comment:
remove the `optional` badge

Reply:
Is this for ELECTRON_START_URL, EVENT_MARKER_PRODUCT_ID, and EVENT_MARKER_COM_NAME?

Reply:
those are optional though, so I think that badge is appropriate, but there shouldn't be any with both optional and default.

Reply:
huh... I think of anything having a default as optional. I think of something queried as throwing an error if missing. And from what I get here, everything is optional

Reply:
If not set, no default is shown (blank input box)  maybe could change to
If not set, user will need to enter the value in the provided input box